modules/opengl/include/modules/opengl/image/imagecompositor.h

```diff
@@ -46,21 +44,13 @@ namespace inviwo {
 class IVW_MODULE_OPENGL_API ImageCompositor {
 public:
     ImageCompositor(std::string programFileName = "composite.frag");
-    virtual ~ImageCompositor() = default;
+    ~ImageCompositor() = default;
 
-    void composite(const Image& source, Image& target, ImageType type);
-    void composite(ImageInport& source, ImageOutport& target, ImageType type);
+    void composite(const Image& source0, const Image& source1, Image& destination, ImageType type);
```

Comment:
Source0 and Source1?

Reply:
the two sources gets composed using depth in to the destination. what that you question?

Reply:
Ah, so before target was also source1?
It would be good if we documented what the class/function does

Reply:
yes, which is undefined behaviour, so we did an internal copy. But in the call site we frist copied one source to the target...